### PR TITLE
Addition of LoginPage 

### PR DESCRIPTION
**Pull request by indra-code**
Added a page which allows the user to choose whether to login or register as a new user

### CODE REVIEW

File: lib/pages/LoginPage.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
```
Comment (anuragrao04) [inline lib/pages/LoginPage.dart:2]: this import fails as this folder doesn't exist. Please add the google_fonts folder. I think you forgot to git add it :)

File: lib/main.dart [hunk 2]
```diff
@@ -34,7 +34,7 @@ class _MyAppState extends State<MyApp> {
       ),
       routes: {
         ""
-        "/": (context) => HomePage(),
+        "/": (context) => LoginPage(),
```
Comment (anuragrao04) [inline lib/main.dart:36]: Please import `LoginPage()` at the top of the file with `import 'package:growpal_hackathon/pages/LoginPage.dart';

**Review by anuragrao04 — CHANGES_REQUESTED** [review]

File: lib/pages/LoginPage.dart (hunk 1 above)
Comment (indra-code) [reply lib/pages/LoginPage.dart:2]: Have updated pubspec.yaml, should work now

**Comment by anuragrao04** [comment]
I have made a few font and color changes, merging now

**Pull request merged**